IDEAS/LIDEAS/Components/LinRectangularZoneTemplate.mo

```diff
@@ -0,0 +1,16 @@
+within IDEAS.LIDEAS.Components;
+model LinRectangularZoneTemplate
+  extends IDEAS.Buildings.Components.RectangularZoneTemplate(redeclare replaceable
+      BaseClasses.LinearAirModel airModel(linearise=linearise),
```

Comment:
I think we can use the `WellMixedAir` model here when it includes `T(stateSelect=StateSelect.prefer)`?